getting-started.md

```diff
@@ -1,35 +1,44 @@
 # Getting Started
 
-## Creating Account
+### Creating Account
 
-You will need the Exogress account to use the service. Please, visit [https://app.exogress.com](https://app.exogress.com/) and create a new account.
+Visit [https://app.exogress.com](https://app.exogress.com/) to create a new account and follow the instructions. Make sure you have activated your account.
 
-## Creating Project
+### Choose your account and subdomain names
 
-Project is the place where your configurations live in. You will be prompted to create your first project once your account is ready. Later, you'll be able to add more projects.
+Custom third-leved subdomain under `exg.link` is provided for your apps and is available by default. You could always add your own domain at any time later.
```

Comment:
It has changed, now use registers subdomain in `<subdomain>.exg.link`, where subdomain is asked during the registration along with account name

Reply:
But that's why it's here under the "Choose your account and subdomain names" section? 
Maybe wording can be better, like:
You will be prompted to provide your account name. This will be associated with your email and couldn't be changed.
Next, we ask you to specify a third-level subdomain under `exg.link`. You could always add your own domain at any time later.

Reply:
actually, both names will be requested from the user when he registers the account - account name and subdomain name.